src/apis/stockx/bids/index.js

```diff
@@ -0,0 +1,136 @@
+import moment from 'moment';
+
+import Api from '../../base';
+import { randomInclusive } from '../../../utils';
+
+export default class BidsApi extends Api {
+  // TODO!
```

Comment:
Will open a follow up issue to implement the remainder of methods. Doesn't necessarily need to be included in this PR.

Reply:
Tracked in https://github.com/matthew1232/stockx-api/issues/4